src/Lti/Model/Lti1p1LaunchUrl.php

```diff
@@ -0,0 +1,48 @@
+<?php
```

Comment:
This class is almost the same as https://github.com/oat-sa/simple-roster/blob/develop/src/Lti/Request/LtiRequest.php (That in my opinion is not a `request`)

Can we have just one of them?